sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb

```diff
@@ -126,13 +127,13 @@
    "source": [
     "from time import gmtime, strftime\n",
     "from sagemaker.model import Model\n",
-    "from sagemaker.amazon.amazon_estimator import get_image_uri\n",
+    "from sagemaker import image_uris\n",
     "\n",
     "model_name = \"DEMO-xgb-churn-pred-model-monitor-\" + strftime(\"%Y-%m-%d-%H-%M-%S\", gmtime())\n",
     "model_url = 'https://{}.s3-{}.amazonaws.com/{}/xgb-churn-prediction-model.tar.gz'.format(bucket, region, prefix)\n",
-    "image_uri = get_image_uri(boto3.Session().region_name, 'xgboost', '0.90-1')\n",
+    "image_uri = sagemaker.image_uris.retrieve('xgboost', boto3.Session().region_name,, '0.90-1')\n",
```

Comment:
Syntax error here. Two commas after `region_name`

Reply:
good catch, thank you very much. I was developing and testing in a corp environment and just 'copy/paste' the code here. I will fix it within a minute

Reply:
No problem! Let me know once you make the fix. Thanks Roman.

Reply:
Hi Paul, I just fixed the typo. Thank you again